src/Nethermind/Nethermind.Synchronization/SnapSync/SnapSyncAllocationStrategyFactory.cs

```diff
@@ -12,7 +12,8 @@ public class SnapSyncAllocationStrategyFactory : StaticPeerAllocationStrategyFac
     {
 
         private static readonly IPeerAllocationStrategy DefaultStrategy =
-            new SatelliteProtocolPeerAllocationStrategy<ISnapSyncPeer>(new TotalDiffStrategy(new BySpeedStrategy(TransferSpeedType.SnapRanges, true), TotalDiffStrategy.TotalDiffSelectionType.CanBeSlightlyWorse), "snap");
+            // TODO: use TotalDiffStrategy in non-merge chains?
```

Comment:
Does not matter for snap sync. In fact, why not just remove the `TotalDiffStrategy` wrapper. It just cause mantainance issue like what you are having now.

Reply:
Wouldn't we need it for any PoW chains we support?

Reply:
Not really, it will just sleep the peer if it fail to return correct snap.